packages/base/src/dialogs/symbology/vector_layer/VectorRendering.tsx

```diff
@@ -93,10 +97,25 @@ const VectorRendering = ({
         RenderComponent = <div>Select a render type</div>;
     }
     setComponentToRender(RenderComponent);
-  }, [selectedRenderType]);
+  }, [selectedRenderType, activeTab]);
 
   return (
     <>
+      {['Single Symbol', 'Graduated', 'Categorized'].includes(
+        selectedRenderType
+      ) && (
+        <div className="jp-gis-symbology-tabs">
+          {['color', 'radius'].map(tab => (
+            <button
+              key={tab}
+              className={`jp-gis-tab ${activeTab === tab ? 'active' : ''}`}
+              onClick={() => setActiveTab(tab as 'color' | 'radius')}
```

Comment:
```suggestion
          {(['color', 'radius'] as const).map(tab => (
            <button
              key={tab}
              className={`jp-gis-tab ${activeTab === tab ? 'active' : ''}`}
              onClick={() => setActiveTab(tab}
```

[const assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions) gives a more specific type so we don't need the cast!

Reply:
Or `<const>['color', 'radius']`